includes/class-sensei-admin.php

```diff
@@ -1431,14 +1431,33 @@ public function handle_order_lessons() {
 			wp_die( esc_html__( 'Insufficient permissions', 'sensei-lms' ) );
 		}
 
-		$course_id = isset( $_POST['course_id'] ) ? (int) $_POST['course_id'] : null;
-		$ordered   = null;
+		if (
+			empty( $_POST['course_id'] )
+			|| empty( $_POST['lessons'] )
+		) {
+			_doing_it_wrong(
+				'handle_order_lessons',
+				'The handle_order_lessons AJAX call should be a POST request with parameters "course_id" and "lessons".',
+				'4.0.1'
+			);
 
-		if ( isset( $_POST['lesson-order'] ) ) {
-			$lesson_order = sanitize_text_field( wp_unslash( $_POST['lesson-order'] ) );
-			$ordered      = $this->save_lesson_order( $lesson_order, $course_id );
```

Comment:
I was not able to use the `save_lesson_order` method because it required major refactoring, which would break backward compatibility and other functionality.